docs/sources/installation/configuration.md

```diff
@@ -78,9 +80,9 @@ system calls to get the machine name.
 
 ### data
 
-Path to where Grafana stores the sqlite3 database (if used), file based
-sessions (if used), and other data.  This path is usually specified via
-command line in the init.d script or the systemd service file.
+Path to where Grafana stores the sqlite3 database (if used), file based sessions (if used), and other data.  This path is usually specified via command line in the init.d script or the systemd service file.
```

Comment:
Sessions is no longer in use

Reply:
So, what should I do with this content? 

Also, should "systemd" be "system.d"?

Reply:
[systemd](https://github.com/systemd/systemd)

Reply:
Sounds good. I saw `init.d` and wondered if the systemd should be the same format.